types/roslib/index.d.ts

```diff
@@ -217,6 +221,21 @@ declare namespace ROSLIB {
          * @param defs - array of type_def dictionary
          */
         decodeTypeDefs(defs: any): void;
+
+        /**
+         * Retrieves a detail of ROS service request.
+         *
+         * @param service name of service:
+         * @param callback - function with params:
+         *   * type - String of the service type
+         * @param failedCallback - the callback function when the service call failed (optional). Params:
+         *   * error - the error message reported by ROS
+         */
+        getServiceRequestDetails(
+            service: string,
+            callback: (type: any) => void,
```

Comment:
Per the above documentation, should `type` be a `string`?
```suggestion
            callback: (type: string) => void,
```